main.go

```diff
@@ -126,38 +128,47 @@ func writeSubnetFile(path string, sn *backend.SubnetDef) error {
 	return os.Rename(tempFile, path)
 }
 
-func lookupIface() (*net.Interface, net.IP, error) {
+func lookupIface() (*net.Interface, net.IP, net.IP, error) {
```

Comment:
Having four-tuple is getting unwieldy. Let's move `eaddr` related logic out to the caller.